packages/react-components/react-portal/src/components/Portal/usePortalMountNode.ts

```diff
@@ -16,6 +16,7 @@ export type UsePortalMountNodeOptions = {
 
 const useStyles = makeStyles({
   root: {
+    position: 'relative',
```

Comment:
What do you think about using [isolation: isolate](https://developer.mozilla.org/en-US/docs/Web/CSS/isolation) here instead?

This seems preferable to me because `isolation` is explicitly for creating a new stacking context and it has [good browser support](https://caniuse.com/?search=isolation).

Reply:
- `position: relative` https://codesandbox.io/s/mystifying-bogdan-diw11s
- `isolation: isolate` https://codesandbox.io/s/wispy-dream-e0iu4g

@spmonahan can you please clarify how it should be used?

Reply:
I think it _should_ be used how you have it in your `isolate` example but clearly that's not working :/

I forked that example to add a couple more stacked divs and from what I can tell anything in an `isolation` stacking context is always stacked below anything in a positioned (i.e., `position: fixed|absolute|etc...`) stacking context. I tested in Edge and Firefox on Windows, iOS Safari and Mac Safari

<img width="203" alt="observed stacking order" src="https://user-images.githubusercontent.com/93940821/173458851-6211bae0-99f6-4799-996b-c1396e1eccbb.png">

https://codesandbox.io/s/laughing-moon-x6n0ws?file=/src/styles.css

Reading through [this Github issue on the CSS WG](https://github.com/w3c/csswg-drafts/issues/2717) that discusses the draw order of stacking contexts created from properties like `isolation` it seems like what we're seeing is the correct behavior.

---
Anyway, I'm fine with using `position: relative` for this.